404.html

```diff
@@ -2,7 +2,42 @@
 <html>
     <head>
         <title>Error 404</title>
-    </head>
+        <style> .announcement-banner { 
+            position: fixed; 
+            top: 0; left: 0; 
+            width: 100%; 
+            background-color: #f8f8f8; 
+            padding: 10px; 
+            text-align: center; 
+            z-index: 9999; 
+        } 
+        .announcement-banner .close-button { 
+            float: right; 
+            cursor: pointer; 
+        } 
+        </style> 
```

Comment:
Should this go in the style.css file if its being used by 2 files?

Reply:
Sure